sweep/sweeper.go

```diff
@@ -98,8 +98,13 @@ type ParamsUpdate struct {
 
 // String returns a human readable interpretation of the sweep parameters.
 func (p Params) String() string {
-	return fmt.Sprintf("fee=%v, force=%v, exclusive_group=%v",
-		p.Fee, p.Force, p.ExclusiveGroup)
+	if p.ExclusiveGroup != nil {
+		return fmt.Sprintf("fee=%v, force=%v, exclusive_group=%v",
+			p.Fee, p.Force, *p.ExclusiveGroup)
```

Comment:
Nit: The exclusive group is (always?) a short channel id, wondering whether it makes sense here to do:

`NewShortChanIDFromInt(*p.ExclusiveGroup)`

You could interpret the exclusive group better maybe ?

Reply:
I think it's fine tho, plus not sure if it's only for scid(tho it seems to be the case). plus if you put it mempool.space you can get the result.